### ISSUE TITLE
About len_in +1

### ISSUE THREAD

**Issue by oliverwang119** [SunQilin/lstm_matlab#6]
Hi Qilin,
Thank you for your code.
I am wondering why there is len_in+1 in x = rand(seq_len, len_in + 1), not len_in?

Thank you,
Oliver

**Comment by QuantumLiu** [comment]
maybe bias

**Comment by SunQilin** [comment]
Yes. It reserves for the bias item. This repository is my first attempt for lstm. Although this code ran correctly, I recommend you use other famous implementations for better performance.
